docs/examples/frameworks/pytorch/pytorch-various-readers.ipynb

```diff
@@ -96,31 +96,26 @@
    "metadata": {},
```

Comment:
<p>I think you can remove <code>ITERATIONS</code>  at all. Now we are reading a small data set, so we don't have to exit early.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done</p>